<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLToolWidget</class>
 <widget class="QWidget" name="SQLToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="sqltool_grid">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="bg_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QSplitter" name="h_splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QGroupBox" name="database_gb">
         <property name="minimumSize">
          <size>
           <width>325</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Database explorer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="connections_cmb">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="disconnect_tb">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Disconnect from all databases</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../res/resources.qrc">
                <normaloff>:/icones/icones/disconnect.png</normaloff>:/icones/icones/disconnect.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QComboBox" name="database_cmb">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAlphabetically</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
              </property>
              <property name="iconSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="refresh_tb">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Update the database list.</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../res/resources.qrc">
                <normaloff>:/icones/icones/atualizar.png</normaloff>:/icones/icones/atualizar.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="databases_tbw">
            <property name="minimumSize">
             <size>
              <width>310</width>
              <height>0</height>
             </size>
            </property>
            <property name="documentMode">
             <bool>false</bool>
            </property>
            <property name="tabsClosable">
             <bool>true</bool>
            </property>
            <property name="movable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="sqlexec_gb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>SQL execution</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QTabWidget" name="sql_exec_tbw">
            <property name="minimumSize">
             <size>
              <width>310</width>
              <height>0</height>
             </size>
            </property>
            <property name="documentMode">
             <bool>false</bool>
            </property>
            <property name="tabsClosable">
             <bool>true</bool>
            </property>
            <property name="movable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>database_cmb</tabstop>
 </tabstops>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
